### ISSUE TITLE
Allow to customize streamlit app

### ISSUE THREAD

**Issue by aguschin** [iterative/mlem#583]
Would be great to allow to customize a bit:
- Change header from "MLEM Streamlit UI"
- Add custom text (e.g. link to the blog post where this model was deployed)

**Comment by aguschin** [comment]
Also, feedback from @omesser: if we're going to add "reshare" button, then the app should be friendly for those who don't know it is. Rn it can be hard to understand, e.g. here you don't know what it is (missing description and header), the current header doesn't tell you anything (what's MLEM?), and you won't get the params unless you know what it is (hide them behind collapsible section?), etc.

<img width="736" alt="image" src="https://user-images.githubusercontent.com/6797716/215678213-ea266deb-dc08-4200-a2d9-2959c207c6c9.png">

**Comment by aguschin** [comment]
@mike0sv, you implemented an option to customize header and description, but it can't be done from CLI. Now CLI usage looks like
```
$ mlem serve streamlit --template custom_template.py
```
I can suggest something like:
```
$ mlem serve streamlit --template.path custom_template.py
$ mlem serve streamlit --template.title "Some title" --template.description "some description"
```

that's a breaking change, but it looks logical to me. The other option is to make it look like simply

```
$ mlem serve streamlit --template custom_template.py
$ mlem serve streamlit --title "Some title" --description "some description"
```
Are you ok with the breaking change or we should implement the alternative?

**Comment by mike0sv** [comment]
This was already done in #602, but I forgot to expose those args to server configuration. Fixed in #607
